### PR TITLE
Update/pages id hero

### PR DESCRIPTION
**Pull request by mjvemo**

### CODE REVIEW

File: lib/components/AwardHero/index.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { selectActiveAward, useSelector } from "@/lib/redux";
+import Link from "next/link";
+import { BreadCrumb } from "primereact/breadcrumb";
+import { Button } from "primereact/button";
+
+export interface ComponentProps {
```
Comment (5up3rD4n1) [inline lib/components/AwardHero/index.tsx:6]: This type of component props is only required for page components, as that's how next.js passes the internal server props to the page component. In this case a simpler component props might be better. Try something like the following:

```ts
export interface ComponentProps {
  id: string;
}

export default function AwardsHero(props: ComponentProps) {
  const {id} = props; 
  ...

  return (...);
}

// in the component usage just pass the id as a prop
...
<AwardsHero id={id}/>
```

**Review by 5up3rD4n1 — CHANGES_REQUESTED** [review]

File: lib/components/AwardHero/index.tsx (hunk 1 above)
Comment (mjvemo) [reply lib/components/AwardHero/index.tsx:6]: Done!

Comment (mjvemo) [reply lib/components/AwardHero/index.tsx:6]: forgot to add id as component prop. Added a new commit with that change

**Review by 5up3rD4n1 — DISMISSED** [review]
LGTM

**Review by 5up3rD4n1 — APPROVED** [review]

**Pull request merged**